controllers/templatesync/template_sync.go

```diff
@@ -312,8 +351,41 @@ func (r *PolicyReconciler) Reconcile(ctx context.Context, request reconcile.Requ
 
 		dependencyFailures := r.processDependencies(ctx, dClient, rMapper, templateDeps, tLogger)
 
-		// fetch resource
-		res := dClient.Resource(rsrc).Namespace(instance.GetNamespace())
+		// Instantiate a dynamic client -- if it's a clusterwide resource, then leave off the namespace
+		var res dynamic.ResourceInterface
+		var isClusterwide bool
+
+		if isGatekeeper {
+			res = dClient.Resource(rsrc)
+			isClusterwide = true
```

Comment:
Would you be able to use `mapping` to determine if it's cluster scoped?

Reply:
Holy moly! I didn't know `Scope` was in there! Okay!